.github/workflows/ci.yaml

```diff
@@ -529,15 +529,7 @@ jobs:
       - name: Locate generated specs
         run: mv /tmp/specs .
       - name: Find difference between OpenAPI specifications
-        run: |
-          docker run -v "$(pwd)/specs:/specs" --rm openapitools/openapi-diff:2.0.1 \
-            /specs/main-spec.json \
-            /specs/current-spec.json \
-            --fail-on-incompatible \
-            --markdown /specs/changes.md \
-            --json /specs/changes.json \
-            --text /specs/changes.txt \
-            --html /specs/changes.html
+        run: echo "TEMPORARY SKIP"
```

Comment:
Either we remember to just edit this or I can merge with admin privileges if we all agree on that. I'd rather forcemerge instead of doing 2 PRs but up to you.

Reply:
That'd be awesome if so. I'll revert this change then.